### PR TITLE
add suites, components and mirror options

### PR DESCRIPTION
**Pull request by asdfugil**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -64,7 +76,7 @@ runs:
     - name: Adding Procusus to PATH
       shell: bash
       run: |
-        PROCURSUS_PATHS=("/opt/procursus/games" "/opt/procursus/sbin" "/opt/procursus/bin")
+        PROCURSUS_PATHS=("/opt/procursus/local/bin" "/opt/procursus/local/sbin" "/opt/procursus/games" "/opt/procursus/sbin" "/opt/procursus/bin")
```
Comment (beer-psi) [inline action.yml]: `$GITHUB_PATH` is like a stack, latest PATH added is the first PATH, so it should be:

```bash
PROCURSUS_PATHS=("/opt/procursus/games" "/opt/procursus/sbin" "/opt/procursus/bin" "/opt/procursus/local/sbin" "/opt/procursus/local/bin")
```
----
While I can theoretically just do
```bash
echo '/opt/procursus/local/bin:/opt/procursus/local/sbin:/opt/procursus/bin:/opt/procursus/sbin:/opt/procursus/games' >> $GITHUB_PATH
```
one goal of this action is for the bootstrap to not break if it is called multiple times, so I resorted to this workaround.

File: action.yml [hunk 2]
```diff
@@ -115,25 +127,26 @@ runs:
...
-          ${{ inputs.cache-path }}/procursus/var/lib/apt/lists/partial \
-          ${{ inputs.cache-path }}/procursus/Library/dpkg/triggers/Lock
-
     - name: Install packages
       shell: bash
       if: inputs.packages
       run: |
         sudo apt-get install -y ${{ inputs.packages }}
         
+    - uses: webiny/action-post-run@2.0.1
```
Comment (beer-psi) [inline action.yml]: Caching before installing packages was intentional, as I wanted the bootstrapped environment to be fresh no matter if it was bootstrapped normally or restored from cache. 

Also, caching after a job has finished will also cache unwanted modifications and potentially cause problems (who knows what happens to `/opt/procursus` after this step finishes?). CI is ideally supposed to have the same state at the start of each run.

**Comment by asdfugil** [comment]
also, issues are disabled

**Comment by beer-psi** [comment]
oops fixed

**Pull request merged**